src/seeders/20200212121323-User.js

```diff
@@ -104,6 +106,29 @@ module.exports = {
         createdAt: new Date(),
         updatedAt: new Date(),
       },
+      {
+        id: '75c34027-a2f0-4b50-808e-0c0169fb074c',
+        firstName: 'peter',
+        lastName: 'Ishimwe',
+        email: 'ishimwe@andela.com',
+        password: bcrypt.hashSync('Bien@BAR789', Number(process.env.passwordHashSalt)),
+        isVerified: true,
+        role: 'manager',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        id: uuid(),
```

Comment:
Is there somewhere we are using this user?

Reply:
I'm using the user in my test